.vscode/extensions.json

```diff
@@ -0,0 +1,9 @@
+{
+  "recommendations": [
+    "dbaeumer.vscode-eslint",
+    "esbenp.prettier-vscode",
+    "pflannery.vscode-versionlens",
+    "eamodio.gitlens"
+  ],
+  "unwantedRecommendations": ["ms-vscode.vscode-typescript-tslint-plugin"]
```

Comment:
If I recall the need for this, then this is meant to override the recommendations coming out of the box from VS Code based on the files being used

Since VS Code has stopped recommending this extension, we don't need to call it out as "unwantedRecommendations"